client/src/components/FormAction/FormAction.scss

```diff
@@ -200,13 +166,17 @@
   &:not(:disabled, .disabled) {
     &:hover,
     &:focus,
-    &.focus,
-    &:active,
-    &.active {
+    &.focus {
       background-color: $btn-secondary-bg;
       border-color: $btn-secondary-bg;
       color: $text-muted;
     }
+    &:active,
+    &.active {
+      background-color: var(--bs-btn-active-bg);
+      border-color: var(--bs-btn-active-border-color);
+      color: var(--bs-btn-active-color);
+    }
```

Comment:
`active` needed to be split out to resolve colours being incorrect (e.g. search (magnifying glass) button in gridfields)